src/test/java/com/iota/iri/service/snapshot/impl/SnapshotMockUtils.java

```diff
@@ -0,0 +1,64 @@
+package com.iota.iri.service.snapshot.impl;
+
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.model.Hash;
+import com.iota.iri.service.snapshot.Snapshot;
+import com.iota.iri.service.snapshot.SnapshotProvider;
+import org.mockito.Mockito;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SnapshotMockUtils {
+    private static final int DEFAULT_MILESTONE_START_INDEX = 70000;
+
+    private static final Hash DEFAULT_GENESIS_HASH = Hash.NULL_HASH;
```

Comment:
why not use `Hash.NULL_HASH` directly?

Reply:
To make it easier to reason about the logic